client/src/components/RandomFactCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Typography, CardContent, Card, CardMedia, Button, CardActions } from '@mui/material';
+
+interface RandomFactProps {
+  fact: string;
+}
+
+const RandomFactCard: React.FC<RandomFactProps> = ({ fact }) => {
+  return (
+    <Card sx={{ display: 'flex', flexDirection: 'column', width: 300, height: 400 }}>
+      <CardMedia
+        component="img"
+        height="150"
+        /* NOTE: The url is appended with a unique query string to
+                 circumvent caching. Since the url is the same 
+                 otherwise, browsers would only generate 1 image. */
+        image={`https://picsum.photos/300/200?${Date.now()}`}
```

Comment:
This should be okay but just to be on the safe side maybe we can stick to using an API like pexels? The don't allow explicit content as it's against their guidelines: https://help.pexels.com/hc/en-us/articles/360042795073-What-are-the-requirements-for-uploading-images-

Ref: https://www.pexels.com/api/

Reply:
Sounds good, should be a quick update 👍🏼

Reply:
Hey @rajivtitus, this eluded me at first but since `pexels` is an API there needs to be a key configured properly for the app to work. If we go forward with this change, we would have to figure out how we would share the key. Or potentially create an initial practice issue of some sort to set up this key similar to `s3/cloudinary` issues. I don't know how much sense that makes though, so I'm unsure if this is a good idea 🤔

Reply:
@ugur-akin Good point! I guess to keep things simple we can just stick with our current implementation

Reply:
How about this, I'll generate some random images and just save their urls into my facts. The idea of random images sounded fun but it doesn't seem worth at this point.

Reply:
That sounds good to me 👍🏽